File: 02 - Data Science/06 - CursoPOOUber/Python/main.py
# from Python.uberX import UberX
from car import Car
from account import Account
from uberX import UberX
from uberPool import UberPool
from user import User

if __name__=="__main__":
    print("Hola Mundo")

    car = UberX("AMS234", Account("Andres Herrera", "ANDA876"), "Chevrolte", "Spark")
    print(vars(car))
    print(vars(car.Driver))

    car2 = UberPool("QWE567", Account("Martha", "MDAN456"), "Chevrolet", "Aveo" )
    print(vars(car2))
    print(vars(car2.Driver))

    user = User("Diego Martinez", "REA789", "diego@hotmail.com", "*****")
    print(vars(user))

    

File: 02 - Data Science/06 - CursoPOOUber/Java/Main.java
package Java;

class Main {
    public static void main(String[] args) {
        System.out.println("Hola Mundo");
        UberX uberX = new UberX("AMQ123", new Account("Andres Herrera", "AND123"), "Chevrolet", "Sonic" );
        //uberX.passenger=4;
        uberX.setPassenger(4);
        uberX.printDataCar(); 
        
        UberVan UberVan = new UberVan("FGH345", new Account("Andres Herrera", "AND123"));  
        UberVan.setPassenger(6);
        UberVan.printDataCar(); 
        // Car car2 = new Car("QWE567", new Account("Andrea Herrera","AND456"));
        // car2.Passengen=3;
        // car2.printDataCar();
    }
}